lib/Interpreter/ValueExtractionSynthesizer.cpp

```diff
@@ -433,10 +466,15 @@ namespace {
     // parent interpreter, but it will fail, because this is a template function.
     // Once the import of template functions becomes supported by clang,
     // this check can be de-activated.
-    if (!m_isChildInterpreter)
-      assert(!R.empty() && "Cannot find cling::runtime::internal::copyArray");
-    m_UnresolvedCopyArray
-      = m_Sema->BuildDeclarationNameExpr(CSS, R, /*ADL*/ false).get();
+    if (!m_isChildInterpreter && R.empty())
+      return VSError(m_Sema, E, "cling::runtime::internal::copyArray");
+
+    m_UnresolvedCopyArray = m_Sema->BuildDeclarationNameExpr(CSS, R, ADL).get();
+    if (!m_UnresolvedCopyArray)
+      return VSError(m_Sema, E, "cling::runtime::internal::copyArray");
```

Comment:
No need for an extra diag; `BuildDeclarationNameExpr()` already diagnoses failures. Please simply `return false` here.